src/pages/components/leadspace.mdx

```diff
@@ -107,70 +127,109 @@ There are two expressive type styles to choose from when using Lead space left-a
-1. **Gradient stop:** color: black (#000000), opacity: 50%, on mobile, gradient overlays the entire image
-1. **Background image:** \*optional, width: 100% of container, size: 320 X 370
+1. **Heading** <br/>Type: $expressive-heading-06, color: $text-01, width: full container (4 col)
+1. **Short copy** \*optional <br/>type: $expressive-heading-03, color: $text-01, width: full container (4 col)
+1. **Button group** <br/>Use the Button group component
+1. **Gradient stop** <br/>color: black (#000000), opacity: 50%, on mobile, gradient overlays the entire image
+1. **Inline image (from background image)** \*optional <br/>Width: 100% of container, 4:3
 
 <br />
 <br />
 
-![$display-01 mobile display](../../images/pattern/leadspace/LeadSpace_Mobile_Display01.jpg)
+![$display-01 mobile display](../../images/pattern/leadspace/leadspace_specs_left_demo_4.jpg)
 
 <Caption>$display-01 mobile display</Caption>
 
 <br />
 
-1. **Heading:** type: $expressive-heading-06, color: $text-01, width: 100% of container (4 columns)
-1. **Short copy:** \*optional, type: $expressive-heading-03, color: $text-01, width: 95% of container (4 columns)
-1. **Button group:** Use the Button group component
-1. **Gradient stop:** color: black (#000000), opacity: 50%, on mobile, gradient overlays the entire image
-1. **Background image:** \*optional, width: 100% of container, size: 320 X 370
+1. **Heading** <br/>Type: $display-01, color: $text-01, width: full container (4 col)
+1. **Short copy** \*optional <br/>$expressive-heading-03, color: $text-01, width: full container (4 col)
+1. **Button group** <br/>Use the Button group component
+1. **Gradient stop** <br/>Color: black (#000000), opacity: 50%, on mobile, gradient overlays the entire image
+1. **Inline image (from background image)** \*optional <br/>Width: 100% of container, 4:3
 
-## Design and functional specifications
+### Lead space centered
+
+![leadspace_specs_dark_centered_demo](../../images/pattern/leadspace/leadspace_specs_dark_centered_aligned_demo.jpg)
+
+<Caption>Example: $display-01 at max breakpoint (1584px) for a Super (640px) lead space</Caption>
+
+<br />
+
+1. **Heading** <br/>Type: $display-01, color: $text-01, width: calc(8col – 32px)
+2. **Short copy** \*optional <br/>$expressive-heading-03, color: $text-01, width: calc(6col – 32px)
+3. **Button group** <br/>Use the Button group component
+4. **Content bottom spacer** <br/>Height: $layout-07 + $layout-06 with background image, <br/>or $layout-07 without background image
+5. **Gradient (applied over image)** <br/>Color: black (#000000), linear: top to bottom, <br/>Opacity: 1 at start, 0.6 at 256px from bottom, 0 at 160px from bottom
+6. **Background image** \*optional <br/>Cover, width: 100% of container for all breakpoints. <br/>Image position: center bottom.
 
-The design specs and functional specs for Lead space can be viewed <a href="https://ibm.box.com/s/d9jey3rlfxqmxu059f0x9dtcx7f5ehuh" target="_blank" rel="noreferrer">here</a>.
```

Comment:
Are we adding functional and design specs to the website now? It looks like we're just duplicating our content model here again. I think we should link out to corresponding files like we do on our other pages.

Reply:
I agree with you, i adjusted the examples.